setup/ansible/roles/user/tasks/main.yml

```diff
@@ -1,6 +1,12 @@
 ---
+- name: check if neofetch exists
+  stat:
+    path: "/home/{{ ansible_user }}/.config/neofetch/rpi-small"
+  register: stat_result
+
 - name: Remove motd
   command: update-rc.d motd remove
+  when: stat_result.stat.exists == false
```

Comment:
I would perhaps stat the .config/neofetch directory, but also I am unsure why we need to do this check here. Did you want to keep the motd if neofetch isn't installed?

Reply:
Stating either a directory or a file is fine, let me know which you'd prefer.
As for why :  ansible-lint wants a `when` block around commands to make them as close to idempotent as possible. Linking the `update-rc.d motd remove` to success state of installing neofetch ( a file or directory) we can tell if it has already ran and not rerun the disable motd section.

Reply:
We should check the neofetch directory since the rpi-small ASCII text was merged into their master branch and will (eventually) be removed from this repo when neofetch is updated.

https://github.com/dylanaraps/neofetch/commit/6b251e7a0c38ef5ff25c9afb1d53b4685f558e2c

Reply:
fixed